src/main/java/org/broadinstitute/dsm/export/WorkflowForES.java

```diff
@@ -0,0 +1,46 @@
+package org.broadinstitute.dsm.export;
+
+import lombok.Data;
+import lombok.NonNull;
+import org.broadinstitute.dsm.db.DDPInstance;
+
+@Data
+public class WorkflowForES {
+    DDPInstance instance;
+    String ddpParticipantId;
+    String workflow;
+    String status;
+    StudySpecificData studySpecificData;
+
+    private WorkflowForES(DDPInstance instance, String ddpParticipantId, String workflow, String status, StudySpecificData studySpecificData) {
+        this.instance = instance;
+        this.ddpParticipantId = ddpParticipantId;
+        this.workflow = workflow;
+        this.status = status;
+        this.studySpecificData = studySpecificData;
+    }
+
+    public static WorkflowForES createInstance(@NonNull DDPInstance instance, @NonNull String ddpParticipantId,
```

Comment:
In my opinion builder is more flexible here instead of having two factory method.